client/src/components/food-list/NutritionLabel.jsx

```diff
@@ -0,0 +1,111 @@
+            calories: data.calories,
+            weight: data.ingredients[0].parsed[0].weight.toFixed(0) + "g",
+            fat:
+              data.totalNutrients.FAT.quantity.toFixed(2) +
+              data.totalNutrients.FAT.unit,
+            satFat:
+              data.totalNutrients.FASAT.quantity.toFixed(2) +
+              data.totalNutrients.FASAT.unit,
+            protein:
+              data.totalNutrients.PROCNT.quantity.toFixed(2) +
+              data.totalNutrients.PROCNT.unit,
+            carbs:
+              data.totalNutrients.CHOCDF.quantity.toFixed(2) +
+              data.totalNutrients.CHOCDF.unit,
+            chole:
+              data.totalNutrients.CHOLE.quantity.toFixed(2) +
+              data.totalNutrients.CHOLE.unit,
+            ingrts: data.ingredients[0].text,
+          }));
+        });
+    };
+    getNutritionLabelData();
+  }, [food]);
+
+  return (
+    <>
+      <div className="nutrition-label-header">
+        {nutritionLabelData.calories && (
+          <>
+            <h2>Nutrition Facts</h2>
+            <p className="nutrition-label-title">
+              Amount per serving
+              <br />
+              <strong>1 serving:</strong> {food} ({nutritionLabelData.weight})
+            </p>
+            <p className="calories">
+              <strong>Calories: </strong>
+              {nutritionLabelData.calories}
+            </p>
+          </>
+        )}
+      </div>
+      <div className="nutrition-body">
+        <p className="nutrition-line">
+          <strong>Fat:</strong> {nutritionLabelData.fat}
+        </p>
+        <p className="nutrition-line">
+          <strong>Saturated Fat:</strong> {nutritionLabelData.satFat}
+        </p>
+        <p className="nutrition-line">
+          <strong>Protein:</strong> {nutritionLabelData.protein}
+        </p>
+        <p className="nutrition-line">
+          <strong>Carbohydrates:</strong> {nutritionLabelData.carbs}
+        </p>
+        <p className="nutrition-line">
+          <strong>Cholesterol:</strong> {nutritionLabelData.chole}
+        </p>
+        <p className="nutrition-line">
+          <strong>Ingredients:</strong> {nutritionLabelData.ingrts}
+        </p>
+      </div>
+      <div className="diet-labels-div nutrition-line">
+        <strong>Diet Labels:</strong>
+        {nutritionLabelData.dietLabels && (
+          <ul>
+            {nutritionLabelData.dietLabels.map((item, ind) => {
+              return <li key={ind}>{item.toLowerCase().replace(/_/, " ")}</li>;
+            })}
+          </ul>
+        )}
+      </div>
+
+      <div className="health-labels-div nutrition-line">
+        <strong>Health Labels:</strong>{" "}
+        <div className="health-labels-list">
+          {" "}
+          {nutritionLabelData.healthLabels && (
+            <ul>
+              {nutritionLabelData.healthLabels.map((item, ind) => {
+                return (
+                  <li key={ind}>{item.toLowerCase().replace(/_/g, " ")}</li>
```

Comment:
Love that you're using `li`s instead of just mapping over `div`s! Way more semantic and accessible!